liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyConfigFileManager.java

```diff
@@ -105,25 +105,17 @@ public static boolean isServerEnvFile(LibertyTextDocument tdi) {
 
     /**
      * Checks if file matches one of these conditions:
-     * - is default server.env file in `src/main/liberty/config`
-     * - is custom env file specified in liberty-plugin-config.xml (generated from
+     * - is named `server.env`
+     * - is custom `<serverEnv>` specified in liberty-plugin-config.xml (generated from
      * build file)
-     * - is default server.env file in custom configDirectory
      * 
      * @param uri - normally comes from LibertyTextDocument.getUri() which is a URI formatted string (file:///path/to/file)
      * @return
      */
     public static boolean isServerEnvFile(String uri) {
         String filePath = normalizeFilePath(uri);
-        if (filePath.endsWith(DEFAULT_SERVER_ENV) || customServerEnvFiles.contains(filePath)) {
-            return true;
-        }
-
-        File file = new File(filePath);
-        if (file.getName().equals("server.env")) {
-            return customConfigDirs.contains(file.getParent());
-        }
-        return false;
+        return filePath.endsWith(SERVER_ENV_FILENAME) || 
```

Comment:
It is better to check the file name for a direct match than `endsWith`, unless you include the path separator (which you need to be careful with on different platforms...do we only have forward slashes because this is a `uri`?).